plugins/chooseYourAdventurePlayer/chooseYourAdventurePlayer.yml

```diff
@@ -0,0 +1,45 @@
+name: Choose Your Own Adventure Player
+description: Plugin to adapt Choose Your Own Adventure Games to the VideoJS player.
+version: 1.0.1
+# requires: CommunityScriptsUILibrary
+settings:
+  accessToken:
+    displayName: Access Token
+    description: Your personal Github access token needed to read private scripts
+    type: STRING
+  gameTag:
+    displayName: Game tag
+    description: Name of the tag indicating a scene is the starting point of game. Player will initialize once scene with this tag is loaded. Only use this tag on the starting scene.
+    type: STRING
+  enableGameSave:
+    displayName: Enable game save
+    description: When enabled you will be giving the opportunity to resume your games from where you last left off which all your choices in tack.
+    type: BOOLEAN
+  resourcePath:
+    displayName: Resource path
+    description: Location where game resources live (images, jsons). It is recommended to create custom served folder for this.
+    type: STRING
+  rootScenesPath:
+    displayName: Root scenes directory path
+    description: Data path where game scenes live.
+    type: STRING
+  useCustomPosters:
+    displayName: Use custom game posters
+    description: When enabled the t=start game screen will use provided poster instead of default scene cover. Poster should be named with the following comvention {resourcePath}/{gameId}/poster_{number}.jpg
+    type: BOOLEAN
+ui:
+  requires:
+    - CommunityScriptsUILibrary
+  javascript:
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/fetchInterceptor.js
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/stashHandler.js
+    - https://cdn.jsdelivr.net/gh/HandyRandyx/stash-plugins@main/utils/registerPathChangeListener.js
+    - https://cdn.jsdelivr.net/gh/cj12312021/stash-plugins@main/utils/waitForClass.js
+    - https://vjs.zencdn.net/8.16.1/video.min.js
+    - https://raw.githubusercontent.com/chrisboustead/videojs-vtt-thumbnails/master/src/plugin.js
+    - cyap-min.js
```

Comment:
Is  `cyap.min.js` not included in the source or linked externally on purpose?

Reply:
my understanding is that they want to limit the plugin to backers but advertise the plugins existence in this repo, the missing files are provided via a private repo